src/test/java/bowling/ScoreTest.java

```diff
@@ -0,0 +1,39 @@
+package bowling;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class ScoreTest {
+
+    @Test
+    void onFrameMaxScoreTest() {
+
+        assertThatThrownBy(
+                () -> Score.of(31)
+        ).isInstanceOf(IllegalArgumentException.class);
+    }
+
+    @Test
+    void onFrameMinScoreTest() {
+
+        assertThatThrownBy(
+                () -> Score.of(-1)
+        ).isInstanceOf(IllegalArgumentException.class);
+    }
+
+    @Test
+    void checkScore() {
+        Score score = Score.of(5);
+
+        assertThat(score.score()).isEqualTo(5);
+    }
+
+    @Test
+    void addCheckScore() {
+        Score score = Score.of(5).add(3);
+
+        assertThat(score.score()).isEqualTo(8);
```

Comment:
```suggestion
        assertThat(score).isEqualTo(Score.of(8));
```

개인적으론 값 객체와 같은 케이스를 비교할 때 getter를 사용하는 것 보단 객체 자체를 사용하여 테스트하는 것을 좋아합니다 ㅎㅎ

Reply:
감사합니다 . 다시그방법으로 테스트해보겠습니다. get이 너무습관화되잇어서 고쳐야겟네요 ㅠㅠ